docs/4.x/tests-github.md

```diff
@@ -0,0 +1,50 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+## Matomo on GitHub Action
```

Comment:
Maybe a better title could be `GitHub Actions we use` or `Overview of GitHub Actions` or something in that direction? I was bit confused by the title